dispatchers.ts

```diff
@@ -21,6 +22,9 @@ export class CommandDispatcher implements ICommandDispatcher {
 				return;
 			}
 
+			this.$logger.trace("System information:");
+			this.$logger.trace(this.$sysInfo.getSysInfo());
```

Comment:
I agree with @ligaz , I've tried the code and getSysInfo is called no matter of the logging level. However if we implement our idea to have log level trace and output it to a file, no matter of the console logging level, such check will not be needed as we'll always need this information.